activity_browser/app/bwutils/commontasks.py

```diff
@@ -130,4 +130,17 @@ def get_default_project_name():
     elif len(bw.projects):
         return next(iter(bw.projects)).name
     else:
-        return None
\ No newline at end of file
+        return None
+
+def get_LCIA_method_name_dict(keys):
+    """LCIA methods in brightway2 are stored in tuples, which is unpractical for display in, e.g. dropdown Menues.
+    Returns a dictionary with
+    key: comma separated string
+    value: brightway2 method tuple
+    """
+    name_tuple_dict = {}
```

Comment:
dict comprehension to make it more readable?
```python
name_tuple_dict = {', '.join(key): key for key in keys}
```

Reply:
It does it now. Thanks for checking! Could you test again?
Two more little things:
- sometimes the plots don't fill the window after recalculating (but when I move the slider in the middle, if fits again)... not sure what is happening here - probably don't have to follow this up now

Reply:
It also relies on Matplotlib 2.2.2 (otherwise tight_layout does not account for the legend in the plot and it gets cut off... ). Shall we say matplotlib>=2.2.2 in the requirements?

Reply:
Will add the mpl version to the requirements